lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -2480,6 +2481,24 @@ def test_get_ora2_responses_already_running(self):
         already_running_status = "An ORA2 responses report generation task is already in progress."
         self.assertIn(already_running_status, response.content)
 
+    def test_collect_course_forums_data_success(self):
+        url = reverse('get_course_forums_usage', kwargs={'course_id': self.course.id.to_deprecated_string()})
+
+        with patch('instructor_task.api.submit_course_forums_usage_task') as mock_submit_ora2_task:
+            mock_submit_ora2_task.return_value = True
```

Comment:
Is this line needed? A very similar test here does not use it: https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor/tests/test_api.py#L2476